interfaces/seccomp/template.go

```diff
@@ -549,3 +549,23 @@ pwritev
 # of socket(), bind(), connect(), etc individually.
 socketcall
 `)
+
+// bindSyscallWorkaround specifies the bind() syscall for systems which
+// don't use AppArmor but only seccomp. In those cases the net package
```

Comment:
I am ok with either way. The only thing we need to ensure, if we decide to only allow seccomp+apparmor, is that we need to enforce that. Right now we offer our users to build with `--disable-apparmor --enable-seccomp`. We should not allow this anymore then to prevent these cases as we can't control how anyone is using our software.